core/common/modules/connect/module.php

```diff
@@ -210,10 +210,11 @@ protected function get_init_settings() {
 		if ( ! $connect ) {
 			return [];
 		}
-		return [
+
+		return apply_filters('elementor/connect/init_settings', [
```

Comment:
Remove from here, move it to filter only the top bar settings

Reply:
Done